src/polyfills.js

```diff
@@ -31,11 +35,13 @@ import {isExperimentOn} from './experiments';
 
 installDOMTokenListToggle(self);
 installFetch(self);
+installKeyboardEventKey(self);
 installMathSign(self);
 installObjectAssign(self);
 installPromise(self);
 installDocContains(self);
 installArrayIncludes(self);
+installArrayFrom(self);
```

Comment:
Can we selectively install these for amp-date-picker? Array.from is already banned in AMP code via conformance-config.textproto. That should alleviate any bundle size bloat concerns.

Reply:
+1 Since these polyfills are only necessary for `amp-date-picker` it would be prudent to only include them there.

Reply:
The Array.from polyfill can move, but keypress events across AMP are broken for IE11 without that shim